tools/tablemerge/tablemerge.xml

```diff
@@ -0,0 +1,158 @@
+<tool id="tablemerge" name="Table Merge" version="1.0.2+galaxy1" profile="24.2">
```

Comment:
this entire file could need a reformat with the galaxy-language-server

Reply:
May I ask what this corresponds to? I am not familiar with galaxy-language-server